plugins/node.d/icinga_hosts

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+##%# family=auto
+
+#%# capabilities=autoconf 
+
+if [ "$1" = "autoconf" ]; then
+    if [ -r /usr/bin/icingacli ]; then
+        echo yes
+        exit 0
+    else
+        echo "no (could not found icingacli)"
+        exit 1
+    fi
+fi
+
+export TERM=vt100
+
+if [ "$1" = "config" ]; then
+
+    echo "graph_title Icinga Hosts"
+    echo 'graph_args --base 1000'
+    echo 'graph_vlabel Count'
+    echo 'graph_category icinga'
+   
+    echo "up.label Up"
+    echo "down.label Down"
+    echo "unreachable.label Unreachable"
+    echo "pending.label Pending"
+
+
+    echo "up.draw AREA"    
+    echo "down.draw STACK"
+    echo "unreachable.draw STACK"
+    echo "pending.draw STACK"
+
+    exit 0
+fi
+
+
+output=$(icingacli monitoring list hosts)
```

Comment:
I would recommend to the json output format combined with a filtering program instead:
```shell
data=$(icingacli monitoring list hosts --format=json)
host_up=$(echo "$data" | jq -r '.[] | select(.host_state != 0) | .host_name' | wc -l)
```

`jq` is used by multiple plugins. Thus it is quite common to depend upon it (to be mentioned in the documentation header and to be checked in the `autoconf` test).